README.md

```diff
@@ -7,6 +7,9 @@
 [![Installer Packaging (CPack)](https://github.com/ROCm/omnitrace/actions/workflows/cpack.yml/badge.svg)](https://github.com/ROCm/omnitrace/actions/workflows/cpack.yml)
 [![Documentation](https://github.com/ROCm/omnitrace/actions/workflows/docs.yml/badge.svg)](https://github.com/ROCm/omnitrace/actions/workflows/docs.yml)
 
+> [!NOTE]
+> Perfetto validation is done with trace_processor v46.0, as there is a known issue with v47. If you are experiencing problems viewing you trace in[Perfetto UI v46.0](https://ui.perfetto.dev/v46.0-35b3d9845/#!/).
```

Comment:
Let's make it very clear by rephrasing the second sentence to,
> If you are experiencing problems viewing your trace in [the latest Perfetto version](https://ui.perfetto.dev), try viewing your trace with [Perfetto v46](https://ui.perfetto.dev/v46.0-35b3d9845/#!/).

Also, is this note sufficient? Or do we have to point them to this from other places in the ROCm docs that mention the default URL? For example, at this reference in [Trace Visualization in the docs](https://github.com/ROCm/omnitrace/blob/main/README.md#trace-visualization), instead of, 
> Visit [ui.perfetto.dev](https://ui.perfetto.dev/) in the web-browser

should we say,

> Visit [ui.perfetto.dev](https://ui.perfetto.dev/) in the web-browser. See [ Note ] ( add pointer here to this note at the beginning of the README ) regarding Perfetto compatibility.

Reply:
Sure. I'll make the change to your top-level note. Let me look at all the documentation.

Reply:
@ROCm/rocm-documentation, what's the best way to link back to this note in the README file from the RST files?